### PR TITLE
Update descriptions for the main object

### PR DESCRIPTION
**Pull request by adamfri**

### CODE REVIEW

File: packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/rich_content.proto [hunk 1]
```diff
@@ -154,9 +154,8 @@ message Decoration {
         MentionData mention_data = 5; // Data for a mention decoration.
         FontSizeData font_size_data = 6; // Data for a font size decoration.
         google.protobuf.Int32Value font_weight_value = 7; // Font weight for a bold decoration.
-        google.protobuf.BoolValue italic_data = 8; // Data for an italic decoration.
-        google.protobuf.BoolValue underline_data = 9; // Data for an underline decoration.
-        SpoilerData spoiler_data = 10; // Data for a spoiler decoration.
```
Comment (lxgreen) [inline packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/rich_content.proto:159]: @adamfri what's the reason of the SpoilerData field removal? We have a requirement that should rely on spoiler data.

**Review by lxgreen — COMMENTED** [review]
please see my comment

File: packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/rich_content.proto (hunk 1 above)
Comment (adamfri) [reply packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/rich_content.proto:159]: Removed by accident. Fixed now.

Comment (lxgreen) [reply packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/rich_content.proto:159]: 👍

**Pull request merged**